api/sendmail/services/sendmail.js

```diff
@@ -0,0 +1,25 @@
+const nodemailer = require('nodemailer');
+
+// Create reusable transporter object using SMTP transport.
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'kuikuigaocn@gmail.com',
```

Comment:
Can we use environment variables for the user and password?

Reply:
Yes, we have to. since we don't want to get user info being exposed.

Reply:
Let's change the code to use env vars and I'll define them.